windows/src/developer/TIKE/oskbuilder/TouchLayoutDefinitions.pas

```diff
@@ -123,12 +123,12 @@   TJSONDef = record
     'rightalt',
     'rightctrl-rightalt',
     //'shift',
-    'shift-lctrl',
-    'shift-rctrl',
-    'shift-lalt',
-    'shift-ralt',
-    'shift-lctrl-lalt',
-    'shift-rctrl-ralt'
+    'leftctrl-shift',
+    'rightctrl-shift',
+    'leftalt-shift',
+    'rightalt-shift',
```

Comment:
A KMW reference for default layer names:

https://github.com/keymanapp/keyman/blob/81d7722bf4840d43589de4bc8a3f632c34ed2574/common/core/web/keyboard-processor/src/keyboards/defaultLayouts.ts#L86-L106

Reply:
So, :+1: